src/core/meshcop/commissioner.hpp

```diff
@@ -78,6 +79,40 @@ class Commissioner
     ThreadError Stop(void);
 
     /**
+     * This method adds a Joiner entry.
+     *
+     * @param[in]  aExtAddress      A reference to the Joiner's extended address.
```

Comment:
Based on my use of Harness (V1.1 R20), 8.1.1 (DUT as Joiner) would ask user to input EUI64 of the Joiner.
If we clear the EUI64 input (nothing input), xEui would be set to -1.

Seems that no case happens for xEui is '*' in R20, then xEui is set to 0xffffffffffffffff as other vendors.